codechecker_lib/arg_handler.py

```diff
@@ -433,14 +428,17 @@ def handle_plist(args):
 
     try:
         items = [(plist, args, context) for plist in os.listdir(args.directory)]
-        pool.map_async(consume_plist, items, 1).get(float('inf'))
+        pool.map_async(consume_plist, items, 1)
```

Comment:
You get an error because lambda funtions can not be pickled (serialized). If you replace the lambda with an out of line function you will not get an error. The get is necessary to actually run the tasks and wait for them being finished.